### PR TITLE
Repair - Disable repair on ERA/Slat armor hitpoints

### PR DESCRIPTION
**Pull request by PabstMirror**
- Disable "miscRepair" action for all era and slat armor hitpoints.
- Can still be repaired via "Full Repair"

ERA is consumable and it doesn't really make sense to be repairable with just a toolkit.
There are multiple hit-points and leaving them in makes interaction cluttered.

ToDo?
FullRepair has steep requirements by default (adv engineer, repair facility)
We could add a "Repair ERA" action, availiable when near a repair vehicle or facility.

### CODE REVIEW

File: addons/repair/stringtable.xml [hunk 1]
```diff
@@ -735,6 +735,38 @@
...
+            <English>Commander Turret</English>
+            <German>Kommandant Turm</German>
+            <Spanish>Comandante Torreta</Spanish>
+            <French>Commander Tourelle</French>
+            <Polish>Dowódca Wieżyczka</Polish>
+            <Czech>Velitel Věž</Czech>
+            <Portuguese>Comandante Torre</Portuguese>
+            <Italian>Comandante Torretta</Italian>
+            <Hungarian>Parancsnok Lövegtorony</Hungarian>
+            <Russian>Командир Башню</Russian>
```
Comment (Dystopian) [inline addons/repair/stringtable.xml]: Башня командира

File: addons/repair/stringtable.xml [hunk 2]
```diff
@@ -735,6 +735,38 @@
...
+            <English>Commander Gun</English>
+            <German>Kommandant Kanone</German>
+            <Spanish>Comandante Cañón</Spanish>
+            <French>Commander Canon</French>
+            <Polish>Dowódca Działo</Polish>
+            <Czech>Velitel Kanón</Czech>
+            <Portuguese>Comandante Canhão</Portuguese>
+            <Italian>Comandante Cannone</Italian>
+            <Hungarian>Parancsnok Ágyú</Hungarian>
+            <Russian>Командир Пушку</Russian>
```
Comment (Dystopian) [inline addons/repair/stringtable.xml]: Пушка командира

File: addons/repair/stringtable.xml [hunk 3]
```diff
@@ -735,6 +735,38 @@
             <Chinesesimp>枪</Chinesesimp>
             <Chinese>槍</Chinese>
         </Key>
+        <Key ID="STR_ACE_Repair_HitComTurret">
```
Comment (Dystopian) [inline addons/repair/stringtable.xml]: Where are this strings used?

**Comment by Drofseh** [comment]
I might call it `Replace ERA` instead of `Repair ERA`.

I feel like ERA panels are hard to put back together after they blow up...

**Pull request merged**